### ISSUE TITLE
couchbase-server-enterprise and couchbase-server-community both don't install

### ISSUE THREAD

**Issue by erg** [caskroom/homebrew-cask#11850]
```
ergmac:~ erg$ brew install Caskroom/cask/couchbase-server-enterprise
==> brew cask install Caskroom/cask/couchbase-server-enterprise
==> Downloading http://packages.couchbase.com/releases/3.0.4/couchbase-server-enterprise_3.0.4-macos_x86_64.zip

curl: (22) The requested URL returned error: 403 Forbidden
Error: Download failed on Cask 'couchbase-server-enterprise' with message: Download failed: http://packages.couchbase.com/releases/3.0.4/couchbase-server-enterprise_3.0.4-macos_x86_64.zip


ergmac:~ erg$ brew install Caskroom/cask/couchbase-server-community
==> brew cask install Caskroom/cask/couchbase-server-community
==> Downloading http://packages.couchbase.com/releases/3.0.4/couchbase-server-community_3.0.4-macos_x86_64.zip

curl: (22) The requested URL returned error: 403 Forbidden
Error: Download failed on Cask 'couchbase-server-community' with message: Download failed: http://packages.couchbase.com/releases/3.0.4/couchbase-server-community_3.0.4-macos_x86_64.zip
```

**Comment by vitorgalvao** [comment]
@Ngrd

**Comment by SeanSith** [comment]
Couchbase 3.0.4 is apparently no longer an official download from upstream. Additionally Couchbase 3.0.3 for MacOS has only been released as Enterprise Edition, and Community Edition is 3.0.1 only at this point (see [here](https://forums.couchbase.com/t/couchbase-community-3-0-download-location/4053)). Appcast URL seems to reflect this situation, as well.

What would would be the best thing to do in this situation? Revert changes to couchbase-server-enterprise back to status [here](https://github.com/caskroom/homebrew-cask/commit/621405993bf6508b53ad0a95681754c5482865bc)? Adjust couchbase-server-community down to 3.0.1 ([previous commit](https://github.com/caskroom/homebrew-cask/commit/ffc5a5268ae0574163d35243039c4abf98820012) shows it went from 2.1.1 to 3.0.4?

**Comment by Ngrd** [comment]
Yes. It seems that they have decided to remove 3.0.4 versions of both Community and Enterprise editions since last update on 7 June 2015 for no reason. Even the appcast has changed and I didn't find any information why they did this.

The best solution here is to revert the changes back to original in #11635 and #11636. The 3.0.4 version is no longer available, so the revert PR are done in #12020 and #12021.
